typescript/infra/config/environments/test/igp.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  ChainMap,
+  GasOracleContractType,
+  OverheadIgpConfig,
+  multisigIsmVerificationCost,
+  objMap,
+} from '@hyperlane-xyz/sdk';
+
+import { TestChains, chainNames } from './chains';
+import { multisigIsm } from './multisigIsm';
+import { owners } from './owners';
+
+function remotes(local: TestChains) {
+  return chainNames.filter((name) => name !== local);
+}
+function getGasOracles(local: TestChains) {
+  return Object.fromEntries(
+    remotes(local).map((name) => [
+      name,
+      GasOracleContractType.StorageGasOracle,
+    ]),
+  );
+}
```

Comment:
It's only a few lines and in the not too distant future they won't be able to be shared (since mainnet will use different oracle types depending on the chain). Okay to leave as is?